htdocs/lib2/logic/useroptions.class.php

```diff
@@ -12,6 +12,7 @@
 
 class useroptions
 {
+
     /** @var int $nUserId */
```

Comment:
Here is another example of inserting a blank line at the start of a class, even before an existing docblock. Looks strange to me, but I am not used to these things. Do we want it this way?